Ghidra/Processors/6805/data/languages/6809.cspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<compiler_spec>
+  <global>
+    <range space="RAM"/>
+  </global>
+  <stackpointer register="S" space="RAM" growth="negative"/>
+  <default_proto>
+    <prototype name="__stdcall" extrapop="-2" stackshift="-2">
+      <input>
+        <pentry minsize="1" maxsize="1">
+          <register name="CC"/>
```

Comment:
So what does the pentry mean? Can these registers be used as a parameter to a function call?
So for me it sounds reasonable to remove CC and DP. DP is not a general purpose register. 
For the U register I am not sure. It could also be used as an alternative stack pointer.

Reply:
Yes the pentry's defines the calling conventions parameters.  If you don't use register they will be mapped in order to the define parameters.

If U is an alternate stack pointer, just like SP it shouldn't be a parameter.  If it is being used as a global stack pointer, then a new .cspec might be needed for the processor.  If it is being used like the EBP frame pointer, then that will just get picked up.  It depends on the code you are reversing and whether a compiler was used with a convention or for an assembly project with defined coding convention of using the U register as a stack pointer.  Currently you can only define one global stack pointer for a stack.